src/Database/Database.php

```diff
@@ -319,7 +319,11 @@ private function logIndex($query)
 		}
 
 		$watchlist = explode(',', $this->configCache->get('system', 'db_log_index_watch'));
-		$blacklist = explode(',', $this->configCache->get('system', 'db_log_index_blacklist'));
+		$blocklist = explode(
+			',',
+			$this->configCache->get('system', 'db_log_index_blocklist') ??
```

Comment:
This is some very internal configuration. I guess no one than me had ever used it. So I would just replace the name.

Reply:
Fair enough, I wanted the update to be seamless.